src/config.template.cjs

```diff
@@ -157,6 +157,13 @@ module.exports = {
 	 */
 	linkCategory: "cross-seed-link",
 
+	/**
+	 * Defines what tag to set on linked torrents
+	 *
+	 * Default is "cross-seed".
+	 */
+	crossSeedTag: "cross-seed",
```

Comment:
I think given that we fallback with commander and allow nullish in Zod that this would be better set to null to imply that the default is set and no changes are being made.

The description is unclear, and given the differences in the client implementations, I think clarity on this and its behavior and effects would be beneficial.